inclusive-design/wecount.inclusivedesign.ca
# Issue-113: Layout footer

## Description

To resolve https://github.com/inclusive-design/wecount.inclusivedesign.ca/issues/113.

Implement [the footer design](https://www.figma.com/file/0lcLol3X5MmOXackT2YbHJ/WeCount-website?node-id=239%3A116)

## Steps to test

1. Run "npm run dev" to start the website and access it at "http://localhost:3000";
2. The footer style should match what's on the design.

## Additional information

As social media accounts for WeCount project haven't been setup, hrefs for social media icons are left empty in this pull request. According to [the IRC discussion with @lliskovoi](http://irc-logs.fluidproject.org/%23fluid-design/%23fluid-design.2020-03-31.log), starting at 2020-03-31T16:36:17, the person who merges this pull request needs to adjust the css to hide the "follow us on social media" section.
